endpoints.json

```diff
@@ -43,5 +43,19 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id": {
```

Comment:
this endpoint should be to post a comment to a particular article - not to post an article. Post requests dont have queries